test/e2e/utils/utils.go

```diff
@@ -36,6 +42,15 @@ func GetEnv(envVar string) string {
 	return env
 }
 
+func GenerateRandomString(n int) string {
```

Comment:
Maybe make it simpler, just a random number between 0 and 1000 and convert to string.